api/repo.go

```diff
@@ -536,6 +571,33 @@ func FindTagPath(exec boil.Executor, id int64) ([]*models.Tag, error) {
 	return ancestors, nil
 }
 
+func FindFileTreeWithOperations(exec boil.Executor, fileID int64) ([]*MFile, error) {
+
+	files := make([]*MFile, 0)
+
+	rsql := fmt.Sprintf(FILES_TREE_WITH_OPERATIONS, fileID, fileID, fileID)
```

Comment:
don't use fmt.Sprintf, queries.Raw takes args slice as last parameter.
It's more idiomatic and more suitable for SQL construction. It does proper escaping of SQL injections and more...

In addition, to avoid repeating the id 3 times you can use $1 in several places in the sql string